### DISCUSSION TITLE
trying to access invetree from the same network

### DISCUSSION THREAD

**Discussion by UPIlobra07** [inventree/InvenTree#6722] [Q&A] [answered]
hello 

im trying to access inventree from a different PC on the same network 
could some one explain how to do that 

thanks

**Comment by SchrodingersGat** [comment, marked as answer]
@UPIlobra07 please provide more information, specifically how have you installed InvenTree? Via docker, or the installer, or some other method?

Assuming docker, then the default host is `http://inventree.localhost` - that is *not available externally*. You will need to work out the IP address of your host computer, and use that, as the `INVENTREE_SITE_URL` e.g. `http://10.0.0.123`

**Comment by UPIlobra07** [comment]
thank you i tested that it works now
